### ISSUE TITLE
Link for requirements in docs not valid

### ISSUE THREAD

**Issue by paradx** [EnMAP-Box/enmap-box#725]
The link for the pip python requirements txt file ( https://raw.githubusercontent.com/EnMAP-Box/enmap-box/main/requirements.txt ) referenced in the latest version of the docs [here](https://enmap-box.readthedocs.io/en/latest/usr_section/usr_installation.html#install-required-python-packages) as well as the v.3.13 gives a 404 error and can therefore not be used.

**Comment by janzandr** [comment]
Yes, the file was moved, but install instructions are now outdated.
For now see here: https://enmap-box.readthedocs.io/en/48-restructure-user-installation-section-by-operating-system/usr_section/usr_installation_new.html

**Comment by janzandr** [comment]
@thielfab would be good, if you could update that on the main branch soon.

**Comment by thielfab** [comment]
Done 👍

**Comment by janzandr** [comment]
@thielfab and @jakimowb, now we only need an update for the MacOS install instructions.
